pkg/operator/operator.go

```diff
@@ -247,6 +248,9 @@ func (o *Operator) Start(ctx context.Context) {
 		wg.Add(1)
 		go func() {
 			defer wg.Done()
+			// Taking the first supported NodeClass to be the default NodeClass
+			ctx = injection.WithNodeClasses(ctx, cp.GetSupportedNodeClasses())
+			ctx = injection.WithClient(ctx, o.GetClient())
```

Comment:
Do we need GetClient() anymore?

Reply:
This was initially in to pull the nodepool hash that would need to be calculated for each version. Tested with the new approach if it would be needed, I will remove it.